src/php/network/inet_pton.js

```diff
@@ -55,18 +55,11 @@ module.exports = function inet_pton (a) { // eslint-disable-line camelcase
         // Invalid IP.
         return false
       }
-      hextet = f(hextet >> 8, hextet & 0xFF)
+      m[j][i] = f(hextet >> 8, hextet & 0xFF)
     }
-    res += m[j].join('')
+    m[j] = m[j].join('')
     res += m[j]
   }
-  x = res.length
-  if (x === 16) {
-    return res
-  } else if (x < 16 && m.length > 1) {
-    return res + '\x00'.repeat(16 - x + 1) + m[1]
-  }
 
-  // Invalid IP
-  return false
+  return m.join('\x00'.repeat(16 - m[0].length - m[1].length))
```

Comment:
`m[1]` can be undefined and reading `.length` of undefined throws error, so let's change that line to

```javascript
  return m.join('\x00'.repeat(16 - m.reduce((tl, m) => tl + m.length, 0)))
```

Reply:
Done! 👍